grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -70,12 +70,13 @@ class OccurrenceTagLib {
             output = fieldName[0..-4].replaceAll("_", " ") + " (range)"
         } else {
 
-            def label = message(code:"facet." + fieldCode, default:fieldName)
+            def label = message(code:"facet." + fieldCode, default:"")
             if (!label){
+                // try without "facet."
                 label = message(code:fieldCode, default:"")
             }
 
-            label = label ?: camelCaseToHuman(text: fieldName)
+            label = label ?: fieldName
```

Comment:
I think we have `camelCaseToHuman` to handle dynamic fields (not DwC, etc. that will never have i18n entries as they are specific to just a single data resource) so that they are easier to read for people. I'm not sure why you removed this function.

Reply:
Yes, it's seems that I'm removing the `cameCaseToHuman` use, but with the current code, this function is not used because of the previous `default: fieldName` code always return something. See my previous conversation in this file. To not modify the current output that as default uses 'fieldName`, I just moved this default part below.
![2020-01-20-01-04-05-screenshot](https://user-images.githubusercontent.com/180085/96520981-8744c400-1270-11eb-9487-1988dfd5c53e.png)

Reply:
Good pickup - agree with that.